### PR TITLE
EP form position details without js

### PR DESCRIPTION
**Pull request by RuslanHamidullin**
EP Form: Position Details without JS version

### CODE REVIEW

File: app/assets/stylesheets/frontend/forms.scss [hunk 1]
```diff
@@ -1251,5 +1251,13 @@ input[type="file"] {
 .non-js-audit-certificate-submit {
   float:right
 }
-// Used for button_to delete links for NON JS cases - end
+
+.view_as_disabled {
+  input, textarea {
+    opacity: 1;
+    color: black;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/frontend/forms.scss [hunk 2]
```diff
@@ -1251,5 +1251,13 @@ input[type="file"] {
 .non-js-audit-certificate-submit {
   float:right
 }
-// Used for button_to delete links for NON JS cases - end
+
+.view_as_disabled {
+  input, textarea {
+    opacity: 1;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Properties should be ordered color, opacity

File: app/assets/stylesheets/frontend/forms.scss [hunk 3]
```diff
@@ -1251,5 +1251,13 @@ input[type="file"] {
 .non-js-audit-certificate-submit {
   float:right
 }
-// Used for button_to delete links for NON JS cases - end
+
+.view_as_disabled {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Selector `view_as_disabled` should be written in lowercase with hyphens

File: app/assets/stylesheets/frontend/forms.scss [hunk 4]
```diff
@@ -1251,5 +1251,13 @@ input[type="file"] {
...
+  input, textarea {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Each selector in a comma sequence should be on its own line

File: app/controllers/form/positions_controller.rb [hunk 5]
```diff
@@ -0,0 +1,105 @@
+class Form::PositionsController < Form::BaseController
+
```
Comment (matthewford) [inline app/controllers/form/positions_controller.rb]: Extra empty line detected at class body beginning.

File: app/controllers/form/positions_controller.rb [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+  def destroy
+    @form_answer.document = remove_position_result_doc
+    @form_answer.save
+
+    redirect_to form_form_answer_positions_url(@form_answer)
+  end
+
+  private
+
+    def position_params
```
Comment (matthewford) [inline app/controllers/form/positions_controller.rb]: Inconsistent indentation detected.

File: app/value_objects/position.rb [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  validates :name, length: { maximum: 500 }
+
+  # Should be 100 words maximum
+  validates :details, length: {
+    maximum: 100,
+    tokenizer: -> (str) { str.split },
+    message: "is too long (maximum is 100 words)"
+  }
+
+  def initialize(attrs={})
```
Comment (matthewford) [inline app/value_objects/position.rb]: Surrounding space missing in default value assignment.

File: app/controllers/form/positions_controller.rb [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+  expose(:created_position_ops) do
+    {
+      "name" => position_params[:name],
+      "details" => position_params[:details],
+      "ongoing" => position_params[:ongoing],
+      "start_month" => position_params[:start_month],
+      "start_year" => position_params[:start_year],
+      "end_month" => position_params[:end_month],
+      "end_year" => position_params[:end_year]
+    }
```
Comment (dreamfall) [inline app/controllers/form/positions_controller.rb]:
```ruby
position_params.select { |k, _| [:name, :details, ...].include?(k) } 
```

**Comment by dreamfall** [comment]
looks good

**Pull request merged**